src/app/forcompanies/findyourtalent/page.tsx

```diff
@@ -59,7 +32,7 @@ const ForCompanies = () => {
         headerText={"What we offer"}
         startingAt={true}
       />
-
+      <HowItWorks howItWorksData={howItWorksData} />
```

Comment:
I made the How it works component on the Find your talents page. A small correction would fix the How it works component on the tranings and project page.

Reply:
> I made the How it works component on the Find your talents page. A small correction would fix the How it works component on the tranings and project page.

I converted that component into a reusable one and fixed it.

Reply:
i think its good thanks